simpleline/render/widgets.py

```diff
@@ -405,3 +360,54 @@ def completed(self):
     def text(self):
         """Contains the description text from the second line."""
         return self._text
+
+
+class ColumnWidget(Widget):
```

Comment:
Can't be removed easily because it is used in the `CenterWidget`.
Add to TODO list for future changes.